clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -937,13 +937,53 @@ check_schedule_file()
     return true;
 }
 
+bool
+check_timestamp_increase_monotonically(void)
```

Comment:
grammar nit: _timstamps_ or _increases_ or _increase_is_monotonic

Reply:
I've changed it to "check_timestamps_increase_monotonically"